sdk/arm/resources/2020-06-01/armresources/examples_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package armresources_test
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/arm/resources/2020-06-01/armresources"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+)
+
+func getDefaultClient() *armresources.Client {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		log.Fatalf("failed to obtain a credential: %v", err)
+	}
+	return armresources.NewDefaultClient(cred, nil)
+}
```

Comment:
I'm thinking we should add this in each sample so users get the whole story about how to use the client and credentials.

Reply:
Agreed, fixed.